### PR TITLE
INT-2595 Create a CD pipeline for the concourse-github-status Docker image

### PR DESCRIPTION
**Pull request by aliculPix4D**
Modify the Dockerfile of https://github.com/Pix4D/concourse-github-status so it can build everything independently

Create a pipeline to CD concourse-github-status

### CODE REVIEW

File: ci/pipelines/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+resources:
+- name: concourse-github-status.git
+  type: git
+  source:
+    uri: git@github.com:Pix4D/concourse-github-status.git
+    private_key: ((concourse_janus_ssh_key))
+    branch: INT-2595-docker_cd
```
Comment (wagdav) [inline ci/pipelines/pipeline.yml]: Don't forget to change this back to master before merging

File: ci/pipelines/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    tag: test
+
+jobs:
+- name: job-build-github-status
+  public: true
+  plan:
+  - get: concourse-github-status.git 
+  - task: build-image
+    file: concourse-github-status.git/ci/tasks/build-image.yml
+    privileged: True
```
Comment (wagdav) [inline ci/pipelines/pipeline.yml]: [question] Why is this required?

Comment (aledeganopix4d) [reply ci/pipelines/pipeline.yml]: Sadly the builder-task README reads: "A stretch goal of this is to support running without privileged: true, though it currently still requires it."
And we've indeed verified that's the case...

File: ci/pipelines/pipeline.yml (hunk 1 above)
Comment (aliculPix4D) [reply ci/pipelines/pipeline.yml]: Changed backed to master.
